Makefile

```diff
@@ -21,23 +21,40 @@ SIMPLENOTE_CHANGES_STD := `find "$(THIS_DIR)" -newer "$(SIMPLENOTE_JS)" \( -name
 SIMPLENOTE_BRANCH = $(shell git --git-dir .git branch | sed -n -e 's/^\* \(.*\)/\1/p')
 
 
-# Build configurations
+##############
+# SIMPLENOTE #
+##############
+
+# Node environment
 NODE_ENV = production
```

Comment:
Would we want this to assign only if we haven't overwritten it from the command line?

```Makefile
NODE_ENV ?= production
```

Reply:
This is actually brilliant as it also enables us to call:
```
NODE_ENV=development make dev
```
in addition to
```
make dev NODE_ENV=development
```

Dennis, 💯 points to you! 😄